src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -995,6 +995,17 @@ public bool IsFrozen
             }
         }
 
+        /// <summary>
+        /// A flag indicating whether the node is in transient mode.
+        /// When a node is in transient mode, the node will not participate in execution,
+        /// Or saved to the graph. It is only used for previewing the AutoComplete cluster in the canvas.
```

Comment:
caps or instead of Or ; maybe AutoComplete result instead of AutoComplete cluster since transient will apply to both single node and cluster

Reply:
Thank you, updated